swagger.json

```diff
@@ -4432,6 +4468,15 @@
             "description": "ID of the asset",
             "type": "integer",
             "format": "int64"
+        },
+        "dataInputIdParam": {
+            "name": "data_input_id",
+            "in": "path",
+            "required": true,
+            "description": "ID of the data_input",
```

Comment:
Nit: data_input is not a variable anywhere, we can just call this "data input".

Reply:
done!